spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/JobExecutionController.java

```diff
@@ -148,8 +148,10 @@ public ResponseEntity<Void> stopJobExecution(
 	@RequestMapping(value = {"/{executionId}"}, method = RequestMethod.PUT, params = "restart=true")
 	@ResponseStatus(HttpStatus.OK)
 	public ResponseEntity<Void> restartJobExecution(
-			@PathVariable("executionId") long jobExecutionId) throws NoSuchJobExecutionException {
-		taskJobService.restartJobExecution(jobExecutionId);
+			@PathVariable("executionId") long jobExecutionId,
+			@RequestParam(value = "useJsonJobParameters", required = false) Boolean useJsonJobParameters
+	) throws NoSuchJobExecutionException {
+		taskJobService.restartJobExecution(jobExecutionId, useJsonJobParameters);
```

Comment:
Sorry, just noticed this on the re-review but should we log at warn (or at the very least debug) the reason of the NSJEE or is this just a standard expected case here? If the latter, maybe a little comment as to why we don't need to log the exception details. Wdyt?

Reply:
Adding a warning log entry.